xml/System.Web.Security/MembershipProvider.xml

```diff
@@ -31,8 +31,12 @@
     <remarks>
       <format type="text/markdown"><![CDATA[  
   
-## Remarks  
- ASP.NET membership is designed to enable you to easily use a number of different membership providers for your ASP.NET applications. You can use the supplied membership providers that are included with the .NET Framework, or you can implement your own provider.  
+## Remarks
+
+> [!NOTE]
+> ASP.NET membership providers have been superseded by [ASP.NET Identity](/aspnet/identity/overview/getting-started/introduction-to-aspnet-identity). You should update your applications to use the ASP.NET Identity platform instead of the built-in membership providers.
```

Comment:
```suggestion
> ASP.NET membership providers have been superseded by [ASP.NET Identity](/aspnet/identity/overview/getting-started/introduction-to-aspnet-identity). We strongly recommend updating apps to use the ASP.NET Identity platform instead of the membership providers.
```